src/app/api/instances/es6/instanceCreate.js

```diff
@@ -0,0 +1,21 @@
+import instanceFormatAsHal from './instanceFormatAsHal';
+import instanceAdded from './instanceAdded';
+import eventStore from '../helpers/eventStoreClient';
+import setTimeout from '../helpers/setTimeout';
+import config from '../../config';
+
+
+export default (req, res) => {
+    let instanceAddedEvent = instanceAdded.create();
+    let args = {
+        name: 'instances',
+        events: instanceAddedEvent
+    };
+    eventStore.postToStream(args)
```

Comment:
We can actually replace the chained thens with async / await :D 

Check out the branches that Denise and Mark are working on for examples.

@MatiasHeffel @pablofreiberg